detail/workload.vue

```diff
@@ -0,0 +1,152 @@
+<script>
+import { get } from '@/utils/object';
+import { STATE, NAME, IMAGE, NODE } from '@/config/table-headers';
+import { POD } from '@/config/types';
+import SortableTable from '@/components/SortableTable';
+import Tabbed from '@/components/Tabbed';
+import Tab from '@/components/Tabbed/Tab';
+import KVTable from '@/components/KVTable';
+
+export default {
+  components: {
+    SortableTable,
+    Tabbed,
+    Tab,
+    KVTable
+  },
+  props:      {
+    id: {
+      type:     String,
+      required: true
+    },
+    namespace: {
+      type:     String,
+      required: true
+    },
+    value: {
+      type:    Object,
+      default: () => {
+        return {};
+      }
+    }
+  },
+  data() {
+    const podHeaders = [STATE, NAME, IMAGE, NODE];
+
+    const volumeHeaders = [
+      {
+        name:      'volumeName',
+        label:     'Name',
+        value:     'name',
+      },
+      {
+        name:  'mountPath',
+        label: 'Mount Path',
+        value: '$.mountPaths[0]'
+      },
+      {
+        name:  'secret',
+        label: 'Secret',
+        value: 'secret.secretName'
+      }
+    ];
+
+    return {
+      volumeHeaders, podHeaders, pods:  []
+    };
+  },
+  computed:   {
+
+    labels() {
+      const { labels = {} } = this.value;
+
+      return labels;
+    },
+
+    annotations() {
+      const { workloadAnnotations = {} } = this.value;
+
+      return workloadAnnotations;
+    },
+
+    podTemplateSpec() {
+      return get(this.value, 'spec.template.spec') || {};
+    },
+
+    volumes() {
+      let { volumes = [] } = this.podTemplateSpec;
+      const { containers = [] } = this.podTemplateSpec;
+      const map = {};
+
+      volumes.forEach((volume) => {
+        map[volume.name] = { ...volume, mountPaths: [] };
+      });
+
+      containers.forEach((container) => {
+        const { volumeMounts = [] } = container;
+
+        volumeMounts.forEach((volumeMount) => {
+          map[volumeMount.name].mountPaths.push(volumeMount.mountPath);
+        });
+      });
+
+      volumes = [];
+      for (const key in map) {
+        volumes.push({ name: key, ...map[key] });
+      }
+
+      return volumes;
+    }
+  },
+  mounted() {
+    this.findPods(this.id, this.namespace);
```

Comment:
This should use `LoadDeps`, `mounted()` doesn't happen on server-side-render so they won't be loaded.